modules/component/src/componentNext.ts

```diff
@@ -420,6 +420,19 @@ export class ComponentNext<P1 extends ComponentProps> {
     )
   }
 
+  list(): iComponentNext<
+    P1,
+    {
+      iState: {[k in number]?: iState<P1>}
+      oState: {[k in number]?: oState<P1>}
+      iActions: Action<iActions<P1>, number>
```

Comment:
- Let it also be able to handle `iActions<P1>` even without a specific address
- if any of the `iActions` come in without an address let it just get broadcast to all the children.

Reply:
What if `.list()` returned a `ListComponent<P1>` instead? How should the types behave in such a case?

Reply:
@tusharmath the typings of `ListComponent` would be the same as what's written here right?